src/scripts/OSFramework/OSUI/Pattern/InlineSvg/InlineSvg.ts

```diff
@@ -0,0 +1,153 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+namespace OSFramework.OSUI.Patterns.InlineSvg {
+	/**
+	 * Defines the interface for OutSystemsUI Patterns
+	 */
+	export class InlineSvg extends AbstractPattern<InlineSvgConfig> implements IInlineSvg {
+		// Store the parent element
+		private _parentSelf: HTMLElement;
+		// Store the platform events
+		private _platformEventOnInitialize: Callbacks.OSInitializedEvent;
+
+		constructor(uniqueId: string, configs: JSON) {
+			super(uniqueId, new InlineSvgConfig(configs));
+		}
+
+		private _setSvgCode(): void {
+			if (this.configs.SVGCode !== '' && !Helper.SVG.IsValid(this.configs.SVGCode)) {
+				this.selfElement.innerHTML = '';
+				console.error('Please provide a valid SVGCode.');
+			} else {
+				this.selfElement.innerHTML = this.configs.SVGCode;
+			}
+		}
+
+		/**
+		 * Sets the A11Y properties when the pattern is built.
+		 *
+		 * @protected
+		 * @memberof OSFramework.Patterns.InlineSvg.InlineSvg
+		 */
+		protected setA11YProperties(): void {
+			console.log(GlobalEnum.WarningMessages.MethodNotImplemented);
+		}
+
+		/**
+		 * Set the callbacks that will be assigned to the pattern
+		 *
+		 * @protected
+		 * @memberof OSFramework.Patterns.InlineSvg.InlineSvg
+		 */
+		protected setCallbacks(): void {
+			console.log(GlobalEnum.WarningMessages.MethodNotImplemented);
+		}
+
+		/**
+		 * Set the html references that will be used to manage the cssClasses and atribute properties
+		 *
+		 * @protected
+		 * @memberof OSFramework.Patterns.InlineSvg.InlineSvg
+		 */
+		protected setHtmlElements(): void {
+			this._parentSelf = Helper.Dom.GetElementById(this.widgetId);
+
+			Helper.AsyncInvocation(this._platformEventOnInitialize, this.widgetId);
```

Comment:
I would trigger this after the this.finishBuild(), on the build() method

Reply:
Fixed